ref-exchange/src/account_deposit.rs

```diff
@@ -5,29 +5,35 @@ use std::convert::TryInto;
 
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::json_types::{ValidAccountId, U128};
-use near_sdk::{assert_one_yocto, env, near_bindgen, AccountId, Balance};
+use near_sdk::{assert_one_yocto, env, near_bindgen, AccountId, Balance, StorageUsage};
 
 use crate::utils::{ext_fungible_token, GAS_FOR_FT_TRANSFER};
 use crate::*;
 
-const MAX_ACCOUNT_LENGTH: u128 = 64;
-const MIN_ACCOUNT_DEPOSIT_LENGTH: u128 = MAX_ACCOUNT_LENGTH + 16 + 4;
+/// Minimum Account Storage used for account registration.
+/// 64 (AccountID bytes) + 2*8 (int32) + byte
+pub const INIT_ACCOUNT_STORAGE: StorageUsage = 64 + 16 + 4;
+/// NEAR native token
+pub const NEAR: AccountId = r#"near"#;
 
 /// Account deposits information and storage cost.
 #[derive(BorshSerialize, BorshDeserialize, Default)]
 #[cfg_attr(feature = "test", derive(Clone))]
 pub struct AccountDeposit {
-    /// Native amount sent to the exchange.
-    /// Used for storage now, but in future can be used for trading as well.
-    pub amount: Balance,
+    /// NEAR sent to the exchange.
+    /// Used for storage and trading.
+    pub ynear: Balance,
     /// Amounts of various tokens in this account.
     pub tokens: HashMap<AccountId, Balance>,
+    pub storage_used: StorageUsage,
 }
 
 impl AccountDeposit {
```

Comment:
how about using a constructor to initialize storage_used? something like this https://github.com/ref-finance/ref-contracts/pull/6/files#diff-445ab9fc382f558a0e9d7831b135b640b79e74be187c7cbdd435b7ecf6e108bbR36

Reply:
Yes, good idea! I have more updates to push.